playbook-website/app/controllers/pages_controller.rb

```diff
@@ -5,6 +5,7 @@
 require "will_paginate/array"
 
 class PagesController < ApplicationController
+  include PlaybookWebsite::PbDocHelper
```

Comment:
Should this be included in the application controller instead? https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/application_controller.rb